spec/spec_helper.rb

```diff
@@ -12,7 +12,7 @@
 end
 
 Slack.configure do |config|
-  config.token = 'token' # ENV['SLACK_API_TOKEN']
+  config.token = '<SLACK_API_TOKEN>' # ENV['SLACK_API_TOKEN']
```

Comment:
This makes it easier for us to filter and real tokens later using VCR's `config.filter_sensitive_data`